### PR TITLE
[CPU] Fixed strides for Pad after optimized Split

### PR DESCRIPTION
**Pull request by a-sidorova**
### Details:
 -  Pad got srcStrides after optimized Split layer which incorrect for dimension gluing(reshaping). In this PR Pad layer creates his own strides by dimensions

### Tickets:
 - 50181

### CODE REVIEW

**Comment by a-sidorova** [comment]
@maxnick could you review please?

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_pad_node.cpp [hunk 1]
```diff
@@ -133,8 +133,15 @@ void MKLDNNPadNode::createPrimitive() {
     params.srcDims = getParentEdgeAt(0)->getBlob()->getTensorDesc().getBlockingDesc().getBlockDims();
     params.dstDims = getChildEdgeAt(0)->getBlob()->getTensorDesc().getBlockingDesc().getBlockDims();
 
-    params.srcStrides = getParentEdgeAt(0)->getBlob()->getTensorDesc().getBlockingDesc().getStrides();
-    params.dstStrides = getChildEdgeAt(0)->getBlob()->getTensorDesc().getBlockingDesc().getStrides();
+    size_t nDims = params.srcDims.size();
+    params.srcStrides.resize(nDims);
+    params.dstStrides.resize(nDims);
+    params.srcStrides[nDims - 1] = 1;
+    params.dstStrides[nDims - 1] = 1;
```
Comment (maxnick) [inline inference-engine/src/mkldnn_plugin/nodes/mkldnn_pad_node.cpp:140]:
```suggestion
    params.srcStrides.resize(nDims, 1);
    params.dstStrides.resize(nDims, 1);
```

**Review by maxnick — APPROVED** [review]

File: inference-engine/src/mkldnn_plugin/nodes/mkldnn_pad_node.cpp (hunk 1 above)
Comment (a-sidorova) [reply inference-engine/src/mkldnn_plugin/nodes/mkldnn_pad_node.cpp]: Changed, thanks

**Comment by a-sidorova** [comment]
@maxnick thank you for review!

@dmitry-gorokhov could you review please?

**Review by dmitry-gorokhov — APPROVED** [review]

**Comment by dmitry-gorokhov** [comment]
@a-sidorova  I am ok to merge the fix as is, but we have to cover this case by subgraph test. Lets do in separate PR asap.

**Pull request merged**